public/index.html

```diff
@@ -5,79 +5,108 @@
+        </div>
+        <ul>
+          <li>
+            <a href="#">Home</a>
+          </li>
+          <li>
+            <a href="/fluent/guide">Documentation</a>
+          </li>
+          <li>
+            <a href="/play">Online Editor</a>
+          </li>
+          <li>
+            <a href="https://github.com/projectfluent/fluent/wiki">Wiki</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+    <div>
+      <header id="intro">
+        <h1>
+          Fluent
         </h1>
-        <nav class="header__nav">
-          <a class="link" href="http://projectfluent.io/fluent/guide">Syntax Guide</a>
-          <a class="link" href="http://projectfluent.io/play">Online Editor</a>
-          <a class="link" href="https://github.com/projectfluent">GitHub</a>
-          <a class="link" href="https://groups.google.com/forum/#!forum/mozilla.tools.l10n">Mailing list</a>
-          <a class="link" href="https://github.com/projectfluent/fluent/wiki">Wiki</a>
-        </nav>
+        <h2>A localization paradigm designed to unleash the expressive power of the natural language.</h2>
+        <a href="https://github.com/projectfluent/fluent/wiki/Integrating-Fluent" class="cta">Get Started</a>
       </header>
-
-      <section class="section section--centered">
-        <p class="pitch">
-
-          <strong class="highlight">Fluent</strong> is a localization paradigm
-          designed to unleash the expressive power of the natural language.
-
+    </div>
+    <div>
+      <main>
+        <ul>
+          <li>
+            <h1>
+              Innovative
+            </h1>
+            <p>Natural-sounding translations that use the grammatical features of each language..</p>
+          </li>
+          <li>
+            <h1>
+              Powerfull & Extensible
+            </h1>
+            <p>
+              Date, time, number, plural rules support. Full bidirectionality.
+              Any internationalization formatters can be added.
+            </p>
+          </li>
+          <li>
+            <h1>
+              Flexible
+            </h1>
+            <p>
+              Can translate on client-side and server-side. Live language switching,
+              robust error handling.
+              JS, Python, Rust and React implementation.
+            </p>
+          </li>
+          <li>
+            <h1>
+              Complete
+            </h1>
+            <p>
+              Visual localization editor, linters, checkers and
+              workflow tooling.
+            </p>
+          </li>
+        </ul>
+        <aside id="demo"></aside>
+      </main>
+    </div>
+    <div>
+      <section id="explainer">
+        <h1>Why are we doing it?</h1>
+        <p>
+        Software localization has been dominated by an outdated paradigm: the translation is just a dictionary of strings which map one-to-one to the English (en-US) copy. This paradigm is unfair and limiting to languages with grammars more complex than English.
```

Comment:
I agree with @Pike that this copy should change. Let's be more positive and avoid words like `unfair`.